### PR TITLE
Fix some escaping issues

### PR DESCRIPTION
**Pull request by cedric-anne**
## Checklist before requesting a review

- [x] I have read the CONTRIBUTING document.
- [x] I have performed a self-review of my code.

## Description

I fixed some issues reported by Psalm.

### CODE REVIEW

File: phpunit/functional/HtmlTest.php [hunk 1]
```diff
@@ -741,9 +741,9 @@ public function testAddConfirmationOnAction()
 
     public function testJsFunctions()
     {
-        $this->assertSame("$('#myid')", \Html::jsGetElementbyID('myid'));
-        $this->assertSame("$('#myid').trigger('setValue', 'myval');", \Html::jsSetDropdownValue('myid', 'myval'));
-        $this->assertSame("$('#myid').val()", \Html::jsGetDropdownValue('myid'));
+        $this->assertSame("$('#myid')", @\Html::jsGetElementbyID('myid'));
```
Comment (trasher) [inline phpunit/functional/HtmlTest.php:744]: Why a `@`?

Comment (cedric-anne) [reply phpunit/functional/HtmlTest.php:744]: Because I deprecate them, but I can handle it differently.

Comment (trasher) [reply phpunit/functional/HtmlTest.php:744]: No, I see depreciation after adding comment, it's OK for the usage.

**Review by trasher — APPROVED** [review]
Please fix CS and tests ;)

**Review by orthagh — APPROVED** [review]

**Pull request merged**